assets/src/infrastructure/ui/inputs/Button/Button.tsx

```diff
@@ -12,7 +12,7 @@ const Button = React.forwardRef<ButtonType, ButtonProps>(({ children, buttonText
 
 	return (
 		<ChakraButton leftIcon={icon} {...props} className={className} ref={ref}>
-			{text}
+			{text && <span>{text}</span>}
```

Comment:
Any specific reason to wrap that in `span`?

Reply:
originally it was Brent's suggestion to be able to have more control in regards to margins, but later it turned out that this was also good because parent font-size property had affected SVG icon and I had targeted font-size only on `span`